lib/process-services/task-list/components/task-header.component.spec.ts

```diff
@@ -141,13 +142,23 @@ describe('TaskHeaderComponent', () => {
             expect(claimButton.nativeElement.innerText).toBe('ADF_TASK_LIST.DETAILS.BUTTON.CLAIM');
         });
 
-        it('should display the claim button to the invovled group', () => {
+        it('should display the claim button if the task has invovled group/people', () => {
             component.taskDetails = new TaskDetailsModel(taskDetailsWithOutAssigneeMock);
             component.ngOnChanges({});
             fixture.detectChanges();
             let claimButton = fixture.debugElement.query(By.css('[data-automation-id="header-claim-button"]'));
             expect(claimButton.nativeElement.innerText).toBe('ADF_TASK_LIST.DETAILS.BUTTON.CLAIM');
         });
+
+        it('should not display the claim/requeue button if the task does not have processInstanceId ', () => {
```

Comment:
Can we change it in 
if the task is not claimable?
In addition can we check, the value of the method isClaimable to not be true?